jdtls.ext/com.microsoft.jdtls.ext.core/src/com/microsoft/jdtls/ext/core/ProjectCommand.java

```diff
@@ -144,7 +156,7 @@ public void acceptSearchMatch(SearchMatch match) {
         SearchEngine searchEngine = new SearchEngine();
         try {
             searchEngine.search(pattern, new SearchParticipant[] {SearchEngine.getDefaultSearchParticipant()},
-                    scope, requestor, null /* progress monitor */);
+                    scope, requestor, new NullProgressMonitor() /* progress monitor */);
```

Comment:
`/* progress monitor */` could be removed since it's not a meaningless param anymore